org.eclipse.cpacep/src/org/eclipse/cpacep/util/FileHandler.java

```diff
@@ -42,4 +41,30 @@ public FileVisitResult visitFileFailed(Path file, IOException exc) throws IOExce
 		Collections.sort(matchedList, String.CASE_INSENSITIVE_ORDER);
 		return matchedList;
 	}
+
+	public static String readFile(String filename) {
```

Comment:
You could use `java.nio.Files#readAllLines` [[1]] for this.

[1]: https://docs.oracle.com/javase/tutorial/essential/io/file.html